src/fa/game_session.py

```diff
@@ -161,3 +171,19 @@ def _exited(self, status):
         self.state = GameSessionState.OFF
         self._logger.info("Game has exited with status code: {}".format(status))
         self.send('GameState', ['Ended'])
+
+        if self._rehost:
+            self.client.host_game(title=self.game_name,
+                                  mod=self.game_mod,
+                                  visibility=self.game_visibility,
+                                  mapname=self.game_map,
+                                  password=self.game_password,
+                                  is_rehost=True)
+
+        self._rehost = False
+        self.game_uid = None
```

Comment:
I guess a dict would be ok, I'm just not a fan of losing the last bit of of compile time checks and auto completion one can get with python x) (it's easy to misspell a dict-key and you have to manually check how the one you need to set is named)